github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -874,4 +874,74 @@ postsubmits:
           resources:
             requests:
               memory: "200Mi"
+    - name: post-project-infra-kubevirt-releasegraph
+      always_run: false
+      annotations:
+        testgrid-create-test-group: "false"
+      cluster: kubevirt-prow-control-plane
+      decorate: true
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+        workdir: true
+      labels:
+        preset-github-credentials: "true"
+      run_if_changed: "robots/cmd/perf-report-creator/shape.yaml"
+      spec:
+        containers:
+        - image: quay.io/kubevirtci/pr-creator:v20250320-f6c439c
+          env:
+          - name: GIMME_GO_VERSION
+            value: "1.22.5"
+          - name: TARGET_GITHUB_REPO
+            value: kubevirt/ci-performance-benchmarks
+          - name: RELEASE_VERSION
+            value: "v1.6.0"
+          - name: SINCE_DATE
+            value: "2025-05-15"
+          - name: BRANCH
+            value: "add-$RELEASE_VERSION"
+          - name: RELEASE_DIR
+            value: "release-$RELEASE_VERSION"
```

Comment:
do we expect to update this job every release?

can we make the release version part of shape.yaml, making this job independent of the release cadence.